library/src/com/google/maps/android/geojson/GeoJsonLineStringStyle.java

```diff
@@ -49,6 +49,15 @@ public void setColor(int color) {
         mPolylineOptions.color(color);
         styleChanged();
     }
+    
+    /**
+     * Sets the GeoJsonLineString clickable
+     *
+     */
+    public void setClickable(boolean clickable){
```

Comment:
Can you also implement [`isClickable`](https://developers.google.com/android/reference/com/google/android/gms/maps/model/PolylineOptions.html#isClickable()) here?